autogen/oai/completion.py

```diff
@@ -102,6 +103,28 @@ class Completion(openai_Completion):
         "prompt": "{prompt}",
     }
 
+    llm_lite_mapping = {
```

Comment:
Wouldn´t it be a good idea to place these mapping into a file named "modelmapping.cfg" and read it at runtime?
Any change to the mapping content would not need any code change! etc.

Reply:
This has been removed, this was me working on ishaan's PR and forgot to take this out after pushing changes. You can resolve this, thanks

Reply:
Don´t able to _resolve_ it (as mentioned by AaronWard) due lacking Collaborator rights. ;-)